src/components/BaseHead.astro

```diff
@@ -1,7 +1,9 @@
 ---
 import Inter from "@fontsource/inter/files/inter-latin-variable-wghtOnly-normal.woff2"
+import "@fontsource/inter/variable.css"
```

Comment:
refactor: moving the CSS loading into the same file where preload headers are added